assets/component-mega-menu.css

```diff
@@ -0,0 +1,67 @@
+.mega-menu {
+  position: static;
+}
+
+.mega-menu__content {
+  background-color: rgb(var(--color-background));
+  border-left: 0;
+  border-radius: 0;
+  border-right: 0;  
+  left: 0;
+  max-height: calc(var(--viewport-height, 100vh) - var(--header-bottom-position, 20rem) - 4rem);
+  overflow-y: auto;
+  padding-bottom: 2.4rem;
+  padding-top: 2.4rem;
+  position: absolute;
+  right: 0;
+  top: 100%;
+  z-index: -1;
+}
+
+.header-wrapper--border-bottom .mega-menu__content {
+  border-top: 0;
+}
+
+.js .mega-menu__content {
+  opacity: 0;
+  transform: translateY(-1.5rem);
+}
+
+.mega-menu[open] .mega-menu__content {
+  animation: animateMenuOpen var(--duration-default) ease;
+  animation-fill-mode: forwards;
+}
+
+@media (prefers-reduced-motion) {
+  .mega-menu[open] .mega-menu__content {
+    opacity: 1;
+    transform: translateY(0);
+  }
+}
+
+.mega-menu__list {
+  display: grid;
+  gap: 2.4rem 4rem;
+  grid-template-columns: repeat(6, minmax(0, 1fr));
+  list-style: none;
+}
+
+.mega-menu__link {
+  color: rgba(var(--color-foreground), 0.75);
+  display: block;
+  line-height: 1.4;
+  padding-bottom: 0.8rem;
+  padding-top: 0.8rem;
```

Comment:
Since this is the same spacing as for the dropdown, I wonder if there can be a way for menu links to share that styling 🤔

Reply:
This is related to https://github.com/Shopify/dawn/pull/1316#discussion_r821844098